hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/context/properties/BootstrapProperties.java

```diff
@@ -573,5 +574,6 @@ public static Function<String, Object> transformFor(String prop) {
                     entry(HEDERA_ALLOWANCES_IS_ENABLED, AS_BOOLEAN),
                     entry(ENTITIES_LIMIT_TOKEN_ASSOCIATIONS, AS_BOOLEAN),
                     entry(UTIL_PRNG_IS_ENABLED, AS_BOOLEAN),
-                    entry(TOKENS_AUTO_CREATIONS_ENABLED, AS_BOOLEAN));
+                    entry(TOKENS_AUTO_CREATIONS_ENABLED, AS_BOOLEAN),
+                    entry(WORKFLOWS_ENABLED, AS_BOOLEAN));
```

Comment:
Imo this property would go more naturally in the `GLOBAL_STATIC_PROPS` section [here](https://github.com/hashgraph/hedera-services/blob/ea43d6109a15944bb92c7a870ae298f29bc017f8/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/context/properties/BootstrapProperties.java#L178) (tying into above comment).